Vex_Competition.c

```diff
@@ -59,84 +42,58 @@ void move_forward(int power,int time)
 
 }
 
-void turn(int power,int time, int direction)//-1 for left, 1 for right
-{
+void turn(int power, int time, string direction) //-1 for left, 1 for right
+ {
+        	int directionNUMB;
+        	if(direction == "Left") { directionNUMB = -1; }
+        	if(direction == "Right") { directionNUMB = 1; }
 
-		motor[FrontLeftDrive] = power*direction;
-		motor[RearLeftDrive] = power*direction;
-		motor[FrontRightDrive] = power*direction*-1;
-		motor[RearRightDrive] = power*direction*-1;
+		motor[FrontLeftDrive] = power*directionNUMB;
+		motor[RearLeftDrive] = power*directionNUMB;
+		motor[FrontRightDrive] = power*directionNUMB*-1;
+		motor[RearRightDrive] = power*directionNUMB*-1;
 
 		wait1Msec(time);
-		motor[FrontLeftDrive] = 0;
-		motor[RearLeftDrive] = 0;
-		motor[FrontRightDrive] = 0;
-		motor[RearRightDrive] = 0;
+		motor[FrontLeftDrive] = vexMotorOff;		//just good practice instead of 0
+		motor[RearLeftDrive] = vexMotorOff;
+		motor[FrontRightDrive] = vexMotorOff;
+		motor[RearRightDrive] = vexMotorOff;
 
 }
 
 void liftarm(int power,int time){
 	motor[LeftArmBottom] = power;
 		motor[LeftArmTop] = power;
-		motor[RightArmBottom] =power;
+		motor[RightArmBottom] = power;
 		motor[RightArmTop] = power;
 	wait1Msec(time);
 
-	motor[LeftArmBottom] = 0;
-		motor[LeftArmTop] = 0;
-		motor[RightArmBottom] = 0;
-		motor[RightArmTop] = 0;
-
-
-
+	motor[LeftArmBottom] = vexMotorOff;
+		motor[LeftArmTop] = vexMotorOff;
+		motor[RightArmBottom] = vexMotorOff;
+		motor[RightArmTop] = vexMotorOff;
 
 }
 
-/////////////////////////////////////////////////////////////////////////////////////////
-//
-//                                 Autonomous Task
-//
-// This task is used to control your robot during the autonomous phase of a VEX Competition.
-// You must modify the code to add your own robot specific commands here.
-//
-/////////////////////////////////////////////////////////////////////////////////////////
-
 task autonomous()
 {
-  // .....................................................................................
-  // Insert user code here.
-  // .....................................................................................
 
 	move_forward(80,500);
+	// quickscopeDemNoob(300 /* Amount */, 1000000 /* AimbotHaxSpeed */);
```

Comment:
You know what I find even more funny, its the 69 lines not shown. xD. Coincidence, I think so :smile: